xla/service/gpu/autotuning/gemm_fusion_autotuner_test.cc

```diff
@@ -883,6 +883,24 @@ ENTRY e {
       [](const TritonGemmConfig& config) { return config.block_k > 16; }));
 }
 
+TEST_F(GemmFusionAutotunerExhaustiveTest, FailingConfigsDoNotAbortCompilation) {
+  CheckTritonAutotuning(R"(
```

Comment:
Does this currently crash?

Reply:
Yes.

Reply:
Could you provide the crash output?

Reply:
```
[ RUN      ] GemmFusionAutotunerExhaustiveTest.FailingConfigsDoNotAbortCompilation
2024-09-10 16:07:22.894095: F xla/service/gpu/autotuning/gemm_fusion_autotuner.cc:846] Non-OK-status: has_executable.status()
Status: INTERNAL: ptxas exited with non-zero error code 65280, output: ptxas fatal   : (C7602) Insufficient registers (128) to compile instruction at line 320 in function c_1. Try to compile with register target of 154 or higher.
ptxas fatal   : Ptx assembly aborted due to errors
: If the error message indicates that a file could not be written, please verify that sufficient filesystem space is provided.Failure occured when compiling fusion c.1 with config '{block_m:128,block_n:512,block_k:32,split_k:1,num_stages:1,num_warps:16,num_ctas:1}'
Fused HLO computation:
%f (a: f16[2048,2048], b: f16[2048,2048]) -> f16[2048,2048] {
  %a = f16[2048,2048]{1,0} parameter(0)
  %b = f16[2048,2048]{1,0} parameter(1)
  ROOT %c = f16[2048,2048]{1,0} dot(f16[2048,2048]{1,0} %a, f16[2048,2048]{1,0} %b), lhs_contracting_dims={1}, rhs_contracting_dims={0}
}
[5 / 6] Testing //xla/service/gpu/autotuning:gemm_fusion_autotuner_test_gpu_h100; 24s local
FAIL: //xla/service/gpu/autotuning:gemm_fusion_autotuner_test_gpu_h100 (see /.../test.log)
Target //xla/service/gpu/autotuning:gemm_fusion_autotuner_test_gpu_h100 up-to-date:
  bazel-bin/xla/service/gpu/autotuning/gemm_fusion_autotuner_test_gpu_h100
INFO: Elapsed time: 35.539s, Critical Path: 34.79s
INFO: 6 processes: 1 internal, 5 local.
INFO: Build completed, 1 test FAILED, 6 total actions
//xla/service/gpu/autotuning:gemm_fusion_autotuner_test_gpu_h100         FAILED in 24.3s
 ```

Reply:
Yeah exactly, those are the ones I was seeing.